packages/ibm-products/src/components/EditTearsheet/EditTearsheet.js

```diff
@@ -50,50 +50,50 @@ export let EditTearsheet = forwardRef(
       title,
       verticalPosition = defaults.verticalPosition,
       onHandleModalClick,
+      onFormChange,
 
       // Collect any other property values passed in.
       ...rest
     },
     ref
   ) => {
     const [currentForm, setCurrentForm] = useState(0);
+    const [formTitles, setFormTitles] = useState([]);
     const contentRef = useRef();
 
-    const handleCurrentForm = (form) => {
-      setCurrentForm(form);
+    const handleCurrentForm = (formIndex) => {
+      setCurrentForm(formIndex);
+      if (onFormChange) {
+        onFormChange(formIndex);
+      }
     };
 
-    const sideNavItems = [
-      { label: 'Topic Name' },
-      { label: 'Location' },
-      { label: 'Partitions' },
-      { label: 'Message retention' },
-    ];
-
-    const influencer = (
-      <div className="tearsheet-stories__dummy-influencer-block">
-        <SideNav
-          aria-label="Side navigation"
-          className={`${blockClass}__side-nav`}
-          expanded={true}
-          isFixedNav={false}
-        >
-          <SideNavItems>
-            {sideNavItems.map((item, index) => {
-              return (
-                <SideNavMenuItem
-                  key={index}
-                  onClick={() => handleCurrentForm(index)}
-                  isActive={currentForm === index}
-                >
-                  {item.label}
-                </SideNavMenuItem>
-              );
-            })}
-          </SideNavItems>
-        </SideNav>
-      </div>
-    );
+    function defaultInfluencer() {
+      return (
+        <div className={`${blockClass}__side-nav-wrapper`}>
+          <SideNav
+            aria-label="Side navigation"
```

Comment:
Can you add a prop for this label so that `Side navigation` isn't hard coded here?